examples/resources/awscc_resiliencehub_resiliency_policy/resiliency_policy.tf

```diff
@@ -0,0 +1,28 @@
+resource "awscc_resiliencehub_resiliency_policy" "example" {
+  policy_name = "example_policy"
+  tier        = "MissionCritical"
+  policy = {
+    software = {
+      rpo_in_secs = 900
+      rto_in_secs = 3600
+    }
+    hardware = {
+      rpo_in_secs = 300
+      rto_in_secs = 300
+    }
+    region = {
+      rpo_in_secs = 300
+      rto_in_secs = 300
+    }
+    az = {
+      rpo_in_secs = 300
+      rto_in_secs = 300
+    }
+  }
+  policy_description       = "This is an example policy"
+  data_location_constraint = "us-west-2"
```

Comment:
Thank you opening this PR @LearningNewbie . Please update the examples based on the constraints some of these inputs have.

Reply:
Updated both data_location_constraint and tags. Applied multiple times to check for idempotency.